README.md

```diff
@@ -481,6 +481,9 @@ arXiv:1805.06140, 2018.
 - <a name="Mostafavi18arxiv"></a>Mostafavi I., S.M., Wang, L., Ho, Y.S., Yoon, K.J.,  
 *[Event-based High Dynamic Range Image and Very High Frame Rate Video Generation using Conditional Generative Adversarial Networks](https://arxiv.org/abs/1811.08230)*,  
 arXiv:1811.08230, 2018.
+- <a name="Pini18arxiv"></a>Pini S., Borghi G., Vezzani R., Cucchiara R.,
+*[Learn to See by Events: RGB Frame Synthesis from Event Cameras](https://arxiv.org/abs/1812.02041)*,
```

Comment:
Please leave two white spaces to force a line break. 
The title and publication venue should appear in two separate lines.